libraries/types/AbiSerializer.cpp

```diff
@@ -4,187 +4,105 @@
+      built_in_types.emplace("FixedString16", packUnpack<FixedString16>());
+      built_in_types.emplace("TypeName",      packUnpack<TypeName>());
+      built_in_types.emplace("Bytes",         packUnpack<Bytes>());
+      built_in_types.emplace("UInt8",         packUnpack<uint8_t>());
+      built_in_types.emplace("UInt16",        packUnpack<uint16_t>());
+      built_in_types.emplace("UInt32",        packUnpack<uint32_t>());
+      built_in_types.emplace("UInt64",        packUnpack<uint64_t>());
+      built_in_types.emplace("UInt128",       packUnpack<UInt128>());
+      built_in_types.emplace("UInt256",       packUnpack<UInt256>());
+      built_in_types.emplace("Int8",          packUnpack<int8_t>());
+      built_in_types.emplace("Int16",         packUnpack<int16_t>());
+      built_in_types.emplace("Int32",         packUnpack<int32_t>());
+      built_in_types.emplace("Int64",         packUnpack<int64_t>());
+      //built_in_types.emplace("Int128",      packUnpack<Int128>());
+      //built_in_types.emplace("Int256",      packUnpack<Int256>());
+      //built_in_types.emplace("uint128_t",   packUnpack<uint128_t>());
+      built_in_types.emplace("Name",          packUnpack<Name>());
+      built_in_types.emplace("Field",         packUnpack<Field>());
+      built_in_types.emplace("Struct",        packUnpack<Struct>());
+      built_in_types.emplace("Fields",        packUnpack<Fields>());
+      
+      //generated.hpp
+      built_in_types.emplace("AccountName",             packUnpack<AccountName>());
+      built_in_types.emplace("PermissionName",          packUnpack<PermissionName>());
+      built_in_types.emplace("FuncName",                packUnpack<FuncName>());
+      built_in_types.emplace("MessageName",             packUnpack<MessageName>());
+      //built_in_types.emplace("TypeName",              packUnpack<TypeName>());
+      built_in_types.emplace("AccountPermission",       packUnpack<AccountPermission>());
+      built_in_types.emplace("Message",                 packUnpack<Message>());
+      built_in_types.emplace("AccountPermissionWeight", packUnpack<AccountPermissionWeight>());
+      built_in_types.emplace("Transaction",             packUnpack<Transaction>());
+      built_in_types.emplace("SignedTransaction",       packUnpack<SignedTransaction>());
+      built_in_types.emplace("KeyPermissionWeight",     packUnpack<KeyPermissionWeight>());
+      built_in_types.emplace("Authority",               packUnpack<Authority>());
+      built_in_types.emplace("BlockchainConfiguration", packUnpack<BlockchainConfiguration>());
+      built_in_types.emplace("TypeDef",                 packUnpack<TypeDef>());
+      built_in_types.emplace("Action",                  packUnpack<Action>());
+      built_in_types.emplace("Table",                   packUnpack<Table>());
+      built_in_types.emplace("Abi",                     packUnpack<Abi>());
+      built_in_types.emplace("transfer",                packUnpack<transfer>());
```

Comment:
Anything from generated.hpp need not be included in builtin types. These are described by an ABI.